packages/injected/src/ariaSnapshot.ts

```diff
@@ -164,15 +165,28 @@ function ariaRef(element: Element, role: string, name: string, options?: { forAI
 
 function toAriaNode(element: Element, options?: { forAI?: boolean, refPrefix?: string }): AriaNode | null {
   if (element.nodeName === 'IFRAME') {
+    const iframe = element as HTMLIFrameElement;
+    const isDirectlyActive = element.ownerDocument.activeElement === element;
+    let containsActiveElement = false;
```

Comment:
You can remove lines 170-178, the iframe element will be active in its document if it contains focused element, so `const isActive = element.ownerDocument.activeElement === element;` is sufficient.

Reply:
Looks like the lines are still not removed.